media/css/protocol/protocol.scss

```diff
@@ -5,6 +5,9 @@
 $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
+// Enable Firefox Sans on Bedrock
+@import 'base/fonts/firefox';
```

Comment:
Please remove all references to Firefox Sans from this PR: https://github.com/mozmeao/infra/issues/1118#issuecomment-498386788

Reply:
Removed.